Sources/swift-doc/Supporting Types/Pages/SidebarPage.swift

```diff
@@ -2,6 +2,7 @@ import SwiftSemantics
 import SwiftDoc
 import CommonMarkBuilder
 import HypertextLiteral
+import struct Foundation.Locale
```

Comment:
In some files the compiler was having some issues with `Protocol` once Foundation was imported, also when I used `SwiftSemantics.Protocol` instead of just `Protocol`, so I've imported only `Foundation.Locale` where the issue was happening